python/engine/PythonEngine.cpp

```diff
@@ -25,9 +25,12 @@ void addToPythonPath(const openstudio::path& includePath) {
   if (!includePath.empty()) {
     PyObject* sys = PyImport_ImportModule("sys");
     PyObject* sysPath = PyObject_GetAttrString(sys, "path");
+    Py_DECREF(sys);  // PyImport_ImportModule returns a new reference, decrement it
```

Comment:
Py_DECREF some stuff

Reply:
I wonder how many instances there are where we miss these decrements. Or we have one that should not be. I bet there are a few remaining.